retrolambda/src/main/java/net/orfjackal/retrolambda/interfaces/ClassAnalyzer.java

```diff
@@ -45,10 +45,16 @@ public void visit(int version, int access, String name, String signature, String
 
             @Override
             public MethodVisitor visitMethod(int access, String name, String desc, String signature, String[] exceptions) {
-                if (isConstructor(name) || isStaticMethod(access)) {
```

Comment:
Having the static methods is the mechanism through which I'm going to implement point 2 on #81. By having a list of the private static methods which are synthetic and looking at invokedynamic's to those methods we can remove the private bit on the static method and skip the accessor.